dask_cuda/dgx.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 from dask.distributed import Nanny, SpecCluster, Scheduler
-from distributed.worker import TOTAL_MEMORY
```

Comment:
@jcrist should we provide some compatibility buffer for this?

Reply:
"No" is a fine answer from my perspective, but I thought I'd mention it

Reply:
I'd say no, this should never have been publicly used.

Reply:
I guess "publicly used" doesn't apply to this repo, since it extends dask/distributed, right? I'm not sure we have an alternative to doing this at the moment, but happy to change it to something safer as well if there is one.

Reply:
No, what you've moved it to should be fine. `TOTAL_MEMORY` in `distributed.worker` was more of an implementation detail, I'd say `distributed.system` should be public enough for dask-affiliated projects.

Reply:
Great, thanks for the feedback @jcrist!